source/material_model/drucker_prager.cc

```diff
@@ -0,0 +1,388 @@
+        {
+          reference_rho              = prm.get_double ("Reference density");
+          reference_T                = prm.get_double ("Reference temperature");
+          reference_eta              = prm.get_double ("Reference viscosity");
+          thermal_k                  = prm.get_double ("Thermal conductivity");
+          reference_specific_heat    = prm.get_double ("Reference specific heat");
+          thermal_alpha              = prm.get_double ("Thermal expansion coefficient");
+          prm.enter_subsection ("Viscosity");
+          {
+            minimum_viscosity        = prm.get_double ("Minimum viscosity");
+            maximum_viscosity        = prm.get_double ("Maximum viscosity");
+            reference_strain_rate    = prm.get_double ("Reference strain rate");
+            // Convert degrees to radians
+            phi                      = prm.get_double ("Angle of internal friction") * numbers::PI/180.0;
+            cohesion                 = prm.get_double ("Cohesion");
+          }
+          prm.leave_subsection();
+        }
+        prm.leave_subsection();
+      }
+      prm.leave_subsection();
+    }
+  }
+}
+
+// explicit instantiations
+namespace aspect
+{
+  namespace MaterialModel
+  {
+    ASPECT_REGISTER_MATERIAL_MODEL(DruckerPrager,
+                                   "drucker prager",
+                                   "A material model that has constant values "
+                                   "for all coefficients but the density and viscosity. The defaults for all "
+                                   "coefficients are chosen to be similar to what is believed to be correct "
+                                   "for Earth's mantle. All of the values that define this model are read "
+                                   "from a section ``Material model/Drucker Prager'' in the input file, see "
+                                   "Section~\\ref{parameters:Material_model/Drucker Prager}."
+                                   "Note that the model does not take into account any dependencies of "
+                                   "material properties on compositional fields. "
+                                   "\n\n"
+                                   "The viscosity is computed according to the Drucker Prager frictional "
+                                   "plasticity criterion (non-associative) based on a user-defined "
+                                   "Internal friction angle $\\phi$ and cohesion C. In 3D: "
+                                   " $\\sigma_y = \\frac{6 C \\cos(\\phi)}{\\sqrt(3) (3+\\sin(\\phi))} + "
+                                   "\\frac{2 P \\sin(\\phi)}{\\sqrt(3) (3+\\sin(\\phi))}$, "
+                                   "where P is the pressure. "
+                                   "See for example Zienkiewicz, O. C., Humpheson, C. and Lewis, R. W. (1975), "
+                                   "G\'{e}otechnique 25, No. 4, 671-689. "
```

Comment:
here the use of `\'{e}` is correct, so no need to change (because the current text is copied into latex). However, you do need a second backslash.